api/v1/routes/superadmin.py

```diff
@@ -0,0 +1,45 @@
+"""
+Superadmin endpoints
+"""
+
+from fastapi import APIRouter, Depends, HTTPException, status
+from typing import Annotated
+from sqlalchemy.orm import Session
+
+# from fastapi.responses import JSONResponse
+from api.utils.success_response import success_response
+from api.db.database import get_db
+from api.v1.models.user import User
+from api.v1.services.user import user_service
+from uuid import UUID
+
+superadmin = APIRouter(prefix="/superadmin", tags=["superadmin"])
+
+
+@superadmin.delete("/users/{user_id}")
```

Comment:
do we need the "superadmin" prefix? 
api/v1/users/id is fine - we could just check the role of the auth user...

Reply:
I was only following the spec in the sheet where we were assigned issues. [here](https://docs.google.com/spreadsheets/d/1WKXJJoHSIkMvPdkh363yO-hAYEaNP69GNLuaHpjnvis/edit?gid=2015743857#gid=2015743857)
`api/v1/users/id` definitely seems fine, and more concise.

I think the only problem with changing it would be the other devs working on superadmin endpoints as well, they would have to follow the same format (i.e removing the `superadmin` prefix).

Reply:
Done